onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -2030,14 +2053,18 @@ def _infer_ZipMap(self, node):
 
     def _infer_Attention(self, node):
         shape = self._get_shape(node, 0)
-        shape_bias = self._get_shape(node, 2)
-        if shape and len(shape) == 3 and shape_bias and len(shape_bias) == 1:
+        shape_weights = self._get_shape(node, 1)
+        shape_bias = self._try_get_shape(node, 2)
```

Comment:
what if shape_bias is None?

Reply:
the hidden_size will be read from weights